charts/ti-key-release-2/templates/NOTES.txt

```diff
@@ -18,5 +18,7 @@ This helm chart installs environment for Trusted Identity
     secrets.refreshSec: {{ index .Values "ti-key-release-1" "secrets" "refreshSec" }}
     jssService.type: {{ .Values.jssService.type }}
 
-Execute test:
-  kubectl create -f examples/myubuntu.yaml -n trusted-identity
+To execute a test, create a new namespace then deploy the test application:
+    kubectl create ns test
+    kubectl label ns test tsi-mutate=enabled
```

Comment:
can you remind me again what this is for? I didn't see any checks in `mutationRequired`.